src/components/views/right_panel/UserInfo.tsx

```diff
@@ -448,7 +448,7 @@ export const UserOptionsSection: React.FC<{
                     const inviter = new MultiInviter(roomId || "");
                     await inviter.invite([member.userId]).then(() => {
                         if (inviter.getCompletionState(member.userId) !== "invited") {
-                            throw new Error(inviter.getErrorText(member.userId));
+                            throw new Error(inviter.getErrorText(member.userId) ?? undefined);
```

Comment:
`new Error(undefined)` isn't very useful.

In https://github.com/matrix-org/matrix-react-sdk/pull/10440, I've updated this to fallback to a better error message,

```suggestion
                            throw new Error(
                                inviter.getErrorText(member.userId) ||
                                    `User (${member.userId}) did not end up as invited but no error was given from the inviter utility`,
                            );
```